src/relay/backend/vm/compiler.cc

```diff
@@ -328,6 +328,9 @@ class VMFunctionCompiler : ExprFunctor<void(const Expr& expr)> {
       case Opcode::Move:
       case Opcode::InvokeClosure:
       case Opcode::DeviceCopy:
+      case Opcode::RefCreate:
+      case Opcode::RefRead:
+      case Opcode::RefWrite:
```

Comment:
RefWrite doesn't have last register, right? Maybe set the `last_register_` to -1?